diff --git a/validitysensor/sensor.py b/validitysensor/sensor.py
index a135641..c41935a 100644
--- a/validitysensor/sensor.py
+++ b/validitysensor/sensor.py
@@ -6,6 +6,7 @@ from enum import Enum
 from hashlib import sha256
 from struct import pack, unpack
 from time import sleep
+from pathlib import Path
 
 from usb import core as usb_core
 
@@ -19,8 +20,10 @@ from .tls import tls
 from .usb import usb, CancelledException
 from .util import assert_status, unhex
 
+data_path = '/var/lib/python-validity'
+
 # TODO: this should be specific to an individual device (system may have more than one sensor)
-calib_data_path = '/usr/share/python-validity/calib-data.bin'
+calib_data_path = f'{data_path}/calib-data.bin'
 
 line_update_type1_devices = [
     0xB5, 0x885, 0xB3, 0x143B, 0x1055, 0xE1, 0x8B1, 0xEA, 0xE4, 0xED, 0x1825, 0x1FF5, 0x199
@@ -262,6 +265,7 @@ class Sensor:
         self.calibrate()
 
     def save(self):
+        Path(data_path).mkdir(parents=True, exist_ok=True)
         with open(calib_data_path, 'wb') as f:
             f.write(self.calib_data)
 
