diff --git a/setup.py b/setup.py
index ec4e3ca..7d7342b 100755
--- a/setup.py
+++ b/setup.py
@@ -6,10 +6,9 @@ setup(name='open-fprintd',
       version='0.6',
       py_modules = [],
       packages=['openfprintd'],
-      scripts=[],
+      scripts=['dbus_service/open-fprintd', 'scripts/open-fprintd-suspend.py', 'scripts/open-fprintd-resume.py'],
       data_files=[
           ('share/dbus-1/system-services/', ['dbus_service/net.reactivated.Fprint.service']),
-          ('share/dbus-1/system.d/', ['dbus_service/net.reactivated.Fprint.conf']),
-          ('lib/open-fprintd/', ['dbus_service/open-fprintd', 'scripts/suspend.py', 'scripts/resume.py']),
+          ('share/dbus-1/system.d/', ['dbus_service/net.reactivated.Fprint.conf'])
       ]
 )
