diff --git a/dbus_service/dbus-service b/dbus_service/dbus-service
index 31e10b7..a6457ae 100755
--- a/dbus_service/dbus-service
+++ b/dbus_service/dbus-service
@@ -191,7 +191,7 @@ class Device(dbus.service.Object):
         return hexlify(tls.app(unhexlify(cmd))).decode()
 
 
-backoff_file = '/usr/share/python-validity/backoff'
+backoff_file = '/tmp/python-validity-backoff'
 
 
 # I don't know how to tell systemd to backoff in case of multiple instance of the same template service, help!
