diff --git a/bin/validity-sensors-firmware b/bin/validity-sensors-firmware
index e585f1c..e263510 100755
--- a/bin/validity-sensors-firmware
+++ b/bin/validity-sensors-firmware
@@ -31,8 +31,9 @@ from usb import core as usb_core
 
 from validitysensor.firmware_tables import FIRMWARE_NAMES, FIRMWARE_URIS
 from validitysensor.usb import SupportedDevices
+from validitysensor.tmpdir import tmpdir
 
-python_validity_data = '/usr/share/python-validity/'
+python_validity_data = tmpdir
 
 
 def download_and_extract_fw(dev_type, fwdir, fwuri=None):
